app/api/[uri]/route.tsx

```diff
@@ -201,8 +205,9 @@ export async function POST(
       const label = body.label;
       // TODO - deal with learning outcomes
       const learningOutcomes = body.learningOutcomes;
-      const isPublic = Boolean(body.public);
-      saveDoc({ docId, imagePath, name: label, isPublic });
+      const isPublic = body.public === "true";
```

Comment:
Is there a better way to transmit a boolean across an api rather than with this string?